src/sentry/static/sentry/app/actionCreators/onboardingTasks.tsx

```diff
@@ -14,7 +14,7 @@ export async function updateOnboardingTask(
   updatedTask: Pick<OnboardingTask, 'task' | 'status' | 'data'>
 ) {
   if (api !== null) {
-    await api.requestPromise(`/organizations/${organization.slug}/onboarding-tasks/`, {
```

Comment:
Do we need to handle the error case?

Reply:
I'm not too worried about it, but we could

Maybe this should actually return the request promise and we can handle it farther up

Reply:
I'm okay letting the error bubble up and catching i where this is used. Probably more appropriate, what do you think @billyvg ?